src/llm/http_llm_calculator.cc

```diff
@@ -730,10 +734,8 @@ std::string HttpLLMCalculator::serializeStreamingChunk(const std::string& chunkR
         }
         writer.EndObject();  // }
     } else if (endpoint == Endpoint::COMPLETIONS) {
-        if (!stop) {
-            writer.String("text");
-            writer.String(chunkResponse.c_str());
-        }
+        writer.String("text");
+        writer.String(chunkResponse.c_str());
```

Comment:
Why do you make this change? If it's needed here isn't it also needed in chat completions endpoint?

Reply:
This was a change to prevent crashes while using [vllm benchmarking script](https://github.com/vllm-project/vllm/blob/main/benchmarks/benchmark_serving.py). I haven't found an example where this would be necessary for the other endpoint, but I guess the other one might also be adjusted for consistency.

Reply:
@dkalinowski do you recall why this condition was added in the first place?

Reply:
For chat completions OpenAI API specifies that we should return `delta: {}`, we should not remove that
For completions it was in fact a bug that we returned empty message

Reply:
Okay @bstrzele in such case this implementation is good.